packages/@okta/vuepress-site/docs/reference/api/roles/index.md

```diff
@@ -3086,25 +3086,35 @@ User permissions are only effective with respect to the Group(s) to which the ad
-| `okta.groups.read`                  | Allows the admin to only read information about Groups and their members in your Okta organization                                  | All Groups, a specific Group                 |
-| `okta.groups.appAssignment.manage`  | Allows the admin to manage a Group's app assignment (also need `okta.apps.assignment.manage` to assign to a specific App)           | All Groups, a specific Group                 |
-| `okta.apps.read`                    | Allows the admin to only read information about Apps and their members in your Okta organization                                    | All Apps, All apps of specific type, a specific App |
-| `okta.apps.manage`                  | Allows the admin to fully manage Apps and their members in your Okta organization                                                   | All Apps, All apps of specific type, a specific App |
-| `okta.apps.assignment.manage`       | Allows the admin to only manage assignment operations of an App in your Okta org                                                    | All Apps, All apps of specific type, a specific App |
-
+| Permission type                         | Description                                                                                                                                           | Applicable resource types                    |
+| :-------------------------------------- | :---------------------------------------------------------------------------------------------------------------------------------------------------- | :------------------------------------------- |
+| `okta.users.manage`                     | Allows the admin to create and manage Users and read all profile and credential information for Users.                                                | All Users, all Users within a specific Group |
+| `okta.users.create`                     | Allows the admin to create Users. If the admin is also scoped to manage a Group, that admin can add the User to the Group on creation and then manage.| All Groups, a specific Group                 |
+| `okta.users.read`                       | Allows the admin to read any User's profile and credential information.                                                                               | All Users, all Users within a specific Group |
+| `okta.users.credentials.manage`         | Allows the admin to manage only credential lifecycle operations for a User.                                                                           | All Users, all Users within a specific Group |
+| `okta.users.credentials.resetFactors`   | Gives your delegated admin the ability to reset users' MFA authenticators.                                                                            | All Users, all Users within a specific Group |
```

Comment:
Nitpick: Shouldn't we keep the same "Allows the admin to" prefix for consistency's sake?

Reply:
Fixed now, thanks!